### PR TITLE
Add flamegraphs to performance report

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):

- Not for changelog (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

### CODE REVIEW

**Comment by nikitamikhaylov** [comment]
image

File: docker/test/performance-comparison/entrypoint.sh [hunk 1]
```diff
@@ -146,3 +146,7 @@ dmesg -T > dmesg.log
     report analyze benchmark metrics
 
 cp compare.log /output
+
+# Upload images to S3 to use them in html page
+mkdir -p /output/images/svg
+cp./*.svg /output/images/svg/
```
Comment (azat) [inline docker/test/performance-comparison/entrypoint.sh:152]:
```suggestion
cp ./*.svg /output/images/svg/
```

(as suggested in https://clickhouse-test-reports.s3.yandex.net/24549/fcd679f7cfd5736e7ff524df4b464086c755cc1d/style_check.html#fail1)

File: docker/test/performance-comparison/compare.sh [hunk 2]
```diff
@@ -1028,14 +1028,17 @@ do
         query_file=$(echo "$query" | cut -c-120 | sed 's/[/	]/_/g')
         echo "$query_file" >> report/query-files.txt
 
+        # All metainformation will be saved in flamegraph title and subtitle directly
+        flamegraph_filename=$(clickhouse-local -q "SELECT generateUUIDv4();")
```
Comment (akuzm) [inline docker/test/performance-comparison/compare.sh:1032]: I still want to be able to open them locally, and this makes it impossible. Is the query text problematic? Let's use test name + query index instead. By the way, what about `query_file` variable, maybe you should just switch it to be test + query index? You can do this using `cut -f1-3`. Also not sure what is `report/query-files.txt`, maybe we should delete it.

Comment (nikitamikhaylov) [reply docker/test/performance-comparison/compare.sh:1032]: How I understand, the problem is in that current name has spaces and we have to encode it. Or use another name. `report/query-files.txt` is used when we generate differential flame graphs.

Comment (akuzm) [reply docker/test/performance-comparison/compare.sh:1032]: Do spaces become a problem when you upload the files to S3? Removing query text should fix this.

**Comment by qoega** [comment]
Let's revive this PR?

**Comment by alexey-milovidov** [comment]
@nikitamikhaylov Is it failed?

**Pull request closed without merging**